mrmustard/lab/abstract/transformation.py

```diff
@@ -300,13 +297,14 @@ def __getitem__(self, items) -> Callable:
     # TODO: use __class_getitem__ for compiler stuff
 
     def __rshift__(self, other: Transformation):
-        r"""
-        Concatenates self with other (other after self).
+        r"""Concatenates self with other (other after self).
+
         If any of the two is a circuit, all the ops in it migrate to the new circuit that is returned.
-        E.g.
-        `circ = Sgate(1.0)[0,1] >> Dgate(0.2)[0] >> BSgate(np.pi/4)[0,1]`
-        Arguments:
+        E.g. :math:`circ = Sgate(1.0)[0,1] >> Dgate(0.2)[0] >> BSgate(np.pi/4)[0,1]`
```

Comment:
Should be code rather than math, no?
```suggestion
        E.g., ``circ = Sgate(1.0)[0,1] >> Dgate(0.2)[0] >> BSgate(np.pi/4)[0,1]``
```